scripts/benchmarks_ci.py

```diff
@@ -40,8 +40,85 @@
 import micro_benchmarks
 
 
+class DotNetPerformanceModes(Action):
+    '''
+    Default: (Currently Tiered)
+
+    NoTiering: (Formerly called full_opt)
+        COMPlus_TieredCompilation=0
+            This includes R2R code, useful for comparison against Default and
+            JitOnly for changes to R2R code or tiering.
+
+    JitOnly: Maybe could be called FullOptJitOnly to make it clear what it does
+        COMPlus_TieredCompilation=0
+        COMPlus_ReadyToRun=0
+            This is JIT-only, useful for comparison against Default and NoTier
+            for changes to R2R code or tiering.
+
+    MinOpt:
+        COMPlus_TieredCompilation=0
+        COMPlus_JITMinOpts=1
+            Uses minopt-JIT for methods that do not have pregenerated code,
+            useful for startup time comparisons in scenario benchmarks that
+            include a startup time measurement (probably not for
+            microbenchmarks), probably not useful for a PR.
+
+    For PRs it is recommended to kick off Default, NoTiering, and JitOnly modes
```

Comment:
I think Default would be enough if kicked off automatically, it would be useful to be able to include NoTiering and JitOnly modes manually depending on the change.